src/services/userService.js

```diff
@@ -12,6 +12,19 @@ class UserService {
   static signUp(user) {
     return httpClient.post('/users', user);
   }
+
+  static editProfile(payload) {
+    return httpClient.put(`https://target-mvd-api.herokuapp.com/api/v1/users/${payload.userId}`, {
```

Comment:
Can we add this URL as an `env` variable?

Reply:
I realized it was actually not necessary here at all, the default http interceptor takes care of adding it to the relative path send in here. Already fixed it!